libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -53,10 +53,8 @@ export class BookSearchComponent implements OnInit {
   }
 
   searchBooks() {
-    if (this.searchForm.value.term) {
-      this.store.dispatch(searchBooks({ term: this.searchTerm }));
-    } else {
-      this.store.dispatch(clearSearch());
-    }
+    (this.searchForm.value.term)
```

Comment:
I don't think the ternary operator is more readable. It is move useful for return or assignment statements.

In this case the if-else block is better, especially since blocks can span more than one LOC.